### PR TITLE
fix the Chinese doc for shard_index api

### PR DESCRIPTION
**Pull request by sandyhouse**
fix the Chinese doc for shard_index api

### CODE REVIEW

File: doc/paddle/api/paddle/fluid/layers/shard_index_cn.rst [hunk 1]
```diff
@@ -5,58 +5,39 @@ shard_index
 
 .. py:function:: paddle.fluid.layers.shard_index(input, index_num, nshards, shard_id, ignore_value=-1)
 
+:alias_main: paddle.shard_index
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/layers/shard_index_cn.rst]: 8-10这三行不需要写...

File: doc/paddle/api/paddle/fluid/layers/shard_index_cn.rst [hunk 2]
```diff
@@ -5,58 +5,39 @@ shard_index
 
 .. py:function:: paddle.fluid.layers.shard_index(input, index_num, nshards, shard_id, ignore_value=-1)
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/layers/shard_index_cn.rst]: 这里需要改下，paddle.shard_index

Comment (sandyhouse) [reply doc/paddle/api/paddle/fluid/layers/shard_index_cn.rst]: done.

File: doc/paddle/api/paddle/fluid/layers/shard_index_cn.rst (hunk 1 above)
Comment (sandyhouse) [reply doc/paddle/api/paddle/fluid/layers/shard_index_cn.rst]: done.

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**